Cloudinary/Classes/Core/CLDConfiguration.swift

```diff
@@ -207,7 +207,18 @@ import Foundation
      - returns:                             A new `CLDConfiguration` instance.
      
      */
-    public init(cloudName: String, apiKey: String? = nil, apiSecret: String? = nil, privateCdn: Bool = false, secure: Bool = false, cdnSubdomain: Bool = false, secureCdnSubdomain: Bool = false, secureDistribution: String? = nil, cname: String? = nil, uploadPrefix: String? = nil) {
+    public init(
+        cloudName: String,
```

Comment:
Note that this change (as well as some other comments/formatting changes) also appear in PR #247.

Reply:
Thats because #247 is the first PR, it handles the prep work for all the rest

Reply:
Understood, however it means we can't both merge this and #247 since they're in conflict. We can only merge this with careful squashing so we don't lose the meaning of #247 .